packages/form-builder/src/App.js

```diff
@@ -70,7 +70,9 @@ function App() {
                         <InterfaceSkeleton
                             header={<Header
                                 saveCallback={saveCallback}
+                                showSecondarySidebar={showSecondarySidebar}
                                 toggleSecondarySidebar={toggleSecondarySidebar}
+                                showSidebar={showSidebar}
```

Comment:
@jonwaldstein This will probably need to be refactored to a context hook for builder state. I'm looking into that for a later change.Thoughts?

Reply:
Yeah i'm thinking there will be an application provider for managing this kind of stuff, what you have is fine for now though.